tools/release/platforms/chocolatey/build.py

```diff
@@ -88,7 +88,7 @@ def write_license_file(original_license):
 
 def write_verification_txt(original_verification_txt, version, timestamp):
     dest = "VERIFICATION.txt"
-    with open(original_verification_txt, "r") as fin, open(dest, "w") as fout:
+    with open(original_verification_txt, "rb") as fin, open(dest, "w") as fout:
```

Comment:
Why do we need binary? Should we instead be specifying the encoding to be utf-8?

Reply:
Yes, That's a better fix. I used "rb" because I tried to fix it on the fly while running docker builder. I'll change this to use endcoding instead. Thanks.